build-common/teamcity-hibernate.cfg.xml

```diff
@@ -15,7 +15,7 @@
 			<property name="format_sql">true</property>
 
 			<!-- This is the System.Data.dll provider for MSSQL Server -->
-			<property name="connection.driver_class">NHibernate.Driver.SqlClientDriver</property>
+			<property name="connection.driver_class">NHibernate.Driver.SqlClientDriver, NHibernate.Driver.SqlClient</property>
```

Comment:
Indeed, that is overridden by default values from nhibernate-properties.xml or specific values defined for the build.